Gulpfile.js

```diff
@@ -12,6 +12,7 @@ const clone = require("gulp-clone");
 const newer = require("gulp-newer");
 const tsc = require("gulp-typescript");
 const tsc_oop = require("./scripts/build/gulp-typescript-oop");
+const { getDirSize } = require("./scripts/build/get-dir-size");

```

Comment:
nit, do not use kebab-case, instead use camel case.

Reply:
Ok I changed it 🔧